nautobot/docs/development/index.md

```diff
@@ -204,7 +204,7 @@ labels will be applied for categorization.
 #### Submitting Pull Requests
 
 * If you're interested in contributing to Nautobot, be sure to check out our
-[getting started](https://Nautobot.readthedocs.io/en/stable/development/getting-started/)
+[getting started](development/getting-started/)
```

Comment:
This should a relative link that can be auto-generated. It should instead just link to `getting-started` since the MD cross-references are done using the file paths to the `.md` files.

Reply:
so like this, correct? `[getting started](getting-started)`

Reply:
Yes, correct. (You can test your changes locally by running `mkdocs serve` and connecting your browser to http://localhost:8001)